cli/compilers/wasm_wrap.js

```diff
@@ -0,0 +1,25 @@
+function base64ToUint8Array(data) {{
```

Comment:
Hmm I think if this file is to have a .js filename extension, then it should be syntactically valid JS.

Rather than using `format!`, which induces these double braces, it would be better to use a template of some sort. Maybe you can just do a substring replacement on `BASE64_ENCODED_WASM` (or something)